package.json

```diff
@@ -55,7 +55,7 @@
 		"bip39": "=2.4.0",
 		"chalk": "=2.3.0",
 		"cli-table2": "=0.2.0",
-		"lisk-js": "LiskHQ/lisk-js#1.0.0-dev-lisky",
+		"lisk-js": "LiskHQ/lisk-js#b18cf68d9e17ad1849ca22c96932c82235f22599",
```

Comment:
We have a branch on `lisk-js` so that we don't have to constantly update commit hashes: we can just update that branch to whatever we need.